### ISSUE TITLE
Reference Error in `vuedoc-loader.js`

### ISSUE THREAD

**Issue by iwoodruff** [vue-styleguidist/vue-styleguidist#309]
`vue-styleguidist/loaders/vuedoc-loader.js` is accessing `message` before it's defined, throwing `ReferenceError: message is not defined` instead of the intended warning `vue-docgen-api cannot parse ${path}: ${error}`

referencing `message` before it's defined:
```
const message =
    `vue-docgen-api cannot parse ${componentPath}: ${err}\n\n` + logger.warn(message)
```
https://github.com/vue-styleguidist/vue-styleguidist/blob/dev/packages/vue-styleguidist/loaders/vuedoc-loader.js#L77

**Comment by JessicaSachs** [comment]
@elevatebart meet @iwoodruff :-)

**Comment by elevatebart** [comment]
I wood say it's a pleasure but I prefer the classic "woooooops" !!

**Comment by elevatebart** [comment]
@iwoodruff thank you for the warning. 

I made a typo and eslint did not catch it.

**Comment by iwoodruff** [comment]
The pleasure is mine! Thanks for addressing this so quickly 😄

**Comment by elevatebart** [comment]
delivered in 3.8.3
